src/components/Nav.js

```diff
@@ -1,8 +1,103 @@
+import { CSCENTERLIST } from './CSCENTERLIST';
 import './Nav.scss';
 
-function Nav() {
-  return <div>nav</div>;
-}
+const Nav = () => {
+  const [listOpen, setlistOpen] = useState(false);
+  const listToggleMenu = () => {
+    setlistOpen(listOpen => !listOpen);
+  };
+  const [purpleSmallUis, setpurpleSmallUis] = useState(true);
+  useEffect(() => {
+    setTimeout(() => {
+      setpurpleSmallUis(false);
+    }, 20000);
+  }, []);
+  useEffect(() => {
+    setTimeout(() => {
+      <img src="https://unsplash.com/photos/nnzkZNYWHaU" alt="테스트" />;
+    }, 60000);
+  }, []);
+  return (
+    <div className="navLocation">
+      <NavLogo />
+      <a href="https://www.google.co.kr">
+        <p className="leftInfo">
+          <span className="Info"> 샛별·택배</span> 배송안내
+        </p>
+      </a>
+      <li className="rightMenu">
+        <Link to="/signup">
+          <li className="rightMenuli">회원가입</li>
+        </Link>
+        <Link to="/">
+          <li className="rightMenuli">로그인</li>
+        </Link>
+
+        <li className="rightMenuli" onClick={() => listToggleMenu()}>
+          고객센터
+          <ul className={listOpen ? 'show-menu' : 'hide-menu'}>
+            {CSCENTERLIST.map(CSList => {
+              return (
+                <li className="cscenterList" key={CSList.key}>
+                  {CSList.content}
+                </li>
+              );
+            })}
+          </ul>
+        </li>
+      </li>
+
+      <div className="categoriesGroup">
+        <img
+          src="https://img.icons8.com/material-outlined/24/000000/menu--v1.png"
+          className="imgCate"
+          alt="categorieBtn"
+        />
+        <span className="categorieAll">전체 카테고리</span>
+        <span className="categorieMenuV">채소</span>
+        <span className="categorieMenuF">과일</span>
+        <span className="categorieMenuM">정육</span>
+        <span className="categorieMenuFi">수산</span>
+        <input
+          type="text"
+          placeholder="검색어를 입력해주세요"
+          className="searchField"
+        />
+        <button className="imgSearch">
+          <img
+            src="https://img.icons8.com/material-outlined/24/000000/search--v1.png"
+            className="imgSearch"
+            alt="searchBtn"
+          />
+        </button>
+        <img
+          src="https://img.icons8.com/ios/50/000000/marker--v1.png"
+          className="imgLocation"
+          alt="btnLocation"
+        />
+        <img
+          src="https://img.icons8.com/ios-glyphs/30/000000/like--v2.png"
+          className="imgHeart"
+          alt="btnHeart"
+        />
+        <button className="imgCarts">
+          <img
+            src="https://img.icons8.com/ios/50/000000/shopping-cart.png"
+            className="imgCart"
+            alt="btnCart"
+          />
+        </button>
+        {purpleSmallUis === true ? (
+          <div className="purpleSmallUi">
+            <p>지금 가입하고 인기상품을 100원에 받아가세요</p>
+          </div>
+        ) : null}
```

Comment:
```suggestion
        {purpleSmallUis && (
          <div className="purpleSmallUi">
            <p>지금 가입하고 인기상품을 100원에 받아가세요</p>
          </div>
        )}
```
삼항연산자를 사용하지 않아도 좋은 케이스네요!